docs/overview.rst

```diff
@@ -0,0 +1,213 @@
+    :align: center
+
+.. note::
+
+    *Kinto* synchronisation was designed and built by the `Mozilla Firefox Sync
+    <https://en.wikipedia.org/wiki/Firefox_Sync>`_ team.
+
+
+.. _comparison:
+
+Comparison with other solutions
+===============================
+
+Before starting to create yet another data storage service, we had a long
+look to the existing solutions, to see if that would make sense to extend
+the community effort rather than re-inventing the wheel.
+
+It appeared that solutions we looked at weren't solving the problems we had,
+especially regarding fine-grained permissions.
+
+Here is a comparison table we put together with existing alternative
+technologies.
+
+===========================  ======  ======  ========  =======  ==============
+Project                      Kinto   Parse   Firebase  CouchDB  Remote-Storage
+---------------------------  ------  ------  --------  -------  --------------
+Fine-grained permissions     ✔       ✔       ✔
+Easy query mechanism         ✔       ✔       ✔         [#]_     [#]_
+Conflict resolution          ✔       ✔       ✔         ✔        ✔ [#]_
+Validation                   ✔       ✔       ✔         ✔
+Revision history                                       ✔
+File storage                         ✔                 ✔        ✔
+Batch/bulk operations        ✔       ✔                 ✔
+Changes stream               [#]_    ✔       ✔         ✔
+Pluggable authentication     ✔                         ✔        [#]_
+Pluggable storage / cache    ✔                                  ✔
+Self-hostable                ✔                         ✔        ✔
+Decentralised discovery      [#]_                               ✔
+Open source                  ✔                         ✔        ✔
+Language                     Python                    Erlang   Node.js [#]_
+===========================  ======  ======  ========  =======  ==============
+
+.. [#] CouchDB uses Map/Reduce as a query mechanism, which isn't easy to
+       understand for newcomers.
+.. [#] Remote Storage allows "ls" on a folder, but items are not sorted or
+       paginated.
+.. [#] Kinto uses the same mechanisms as Remote storage for conflict handling.
+.. [#] Notifications support is currently in the work.
+.. [#] Remote Storage supports OAuth2.0 implicit grant flow.
+.. [#] Support for decentralised discovery
+       `is planned <https://github.com/Kinto/kinto/issues/125>`_ but not
+       implemented yet.
+.. [#] Remote Storage doesn't define any default implementation (as it is
+       a procol) but makes it easy to start with JavaScript and Node.js.
+
+You can also read `a longer explanation of our choices and motivations behind the
+creation of Kinto <http://www.servicedenuages.fr/en/generic-storage-ecosystem>`_
+on our blog.
+
+
+.. _FAQ:
+
+FAQ
+===
+
+- How does Kinto compares to CouchDB / Remote Storage?
+
+    Before starting to create yet another data storage service, we had a long
```

Comment:
The answer to this question occurs literally 50 lines above - no need to repeat it here. Suggesting removing this Q/A entirely.